src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1995,6 +1995,9 @@ module T = struct
                               , `String
                                   "Snark fee insufficient to create snark \
                                    worker account" )
+                            ; ( "interrupt_get_completed_work_work_ids"
```

Comment:
duplicated `_work_`  word and in line 2018

Reply:
Good catch